app/views/students/registrations/_register.html.erb

```diff
@@ -17,15 +17,41 @@
   </select>
 </div>
 
-<div class="field">
+<!-- <div class="field">
```

Comment:
Please do not comment this out. We need to take the hall of the student, when he/she fills up the registration form.